source/_patterns/01-molecules/navigation/main-menu.yaml

```diff
@@ -13,6 +15,14 @@ properties:
           url:
             type: string
             minLength: 1
+          hasBorderBottom:
```

Comment:
Have submitted a PR to adjust approach on this: https://github.com/elifesciences/pattern-library/pull/1172